docs/reference/workflows/ecs/concepts/system-update-order.md

```diff
@@ -1,40 +1,47 @@
 <%(TOC)%>
 
-# System update order
+# ECS: System update order
 
-_This document relates to the [ECS workflow]({{urlRoot}}/reference/workflows/overview)._
+The SpatialOS GDK for Unity (GDK) defines several update groups which run relative to [`PlayerLoop.Update`](https://docs.unity3d.com/ScriptReference/Experimental.LowLevel.PlayerLoop.html). Most of your systems should belong to one of these groups to ensure that the system is run in the right order relative to GDK receive and send systems.
```

Comment:
probably all systems should be in there right? as long as they interact with spatialos

Reply:
I said "most" in case they have systems that don't interact with spos / want to use `FixedUpdate`